content/components/action-list.mdx

```diff
@@ -0,0 +1,157 @@
+<ButtonOutline mr={1}>ViewComponent</ButtonOutline>
+<ButtonOutline>Figma component</ButtonOutline>
+-->
+
+## Overview
+
+Action lists can have many applications. They’re the foundation of menus, selection panels, and other overlay components; they can be applied to page sidebars for showing individual actions, for handling local navigation, and for displaying metadata.
+
+Action lists support section dividers and headers for grouping items, and indivudal item dividers for added clarity.
+
+Action lists use a mobile-friendly inset style. Their sizes are adapted on touch devices, and their single-column format should render consistently in any screen size.
+
+Except for read-only and disabled exceptions, action list items are interactive.
+
+
+<!--
+<img alt="Action list wrapper structure" src="https://user-images.githubusercontent.com/223966/118818389-2e8fe500-b8ac-11eb-81b2-8a61f4b3d7ba.png" />
+<Caption>Action list wrapper structure</Caption>
+
+- **Action list item:** An action list item is a generic interactive element, which includes a main text and optional icons, descriptions, auxiliary information, and more.
+- **Action list section:** An action list section uses the ARIA role property to group elements from a list. Depending on the provided values, it renders a divider, a header, or both.
+- **Action list divider:** …
+- **Action list section header:** …
+
+<img alt="Action list item structure" src="https://user-images.githubusercontent.com/223966/118818435-3bacd400-b8ac-11eb-8691-ab9cbba73249.png" />
+<Caption>Action list item structure</Caption>
+
+
+
+## Application
+
+### In overlays
+
+### In pages
+
+
+-->
+
+## Anatomy 
+
+<img src="https://user-images.githubusercontent.com/293280/122128726-d7692b80-cde9-11eb-8e5b-2f4e2eb4b4a1.png" alt="Action list anatomy" />
+
+<img src="https://user-images.githubusercontent.com/293280/121974972-7718b200-cd35-11eb-8c04-1fffbc5deedc.png" alt="Action list item anatomy" />
+
+
+## Options
+
+<Grid gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={5}>
+    <div>
+        <Flex justifyContent="center">
+            <img width="464" alt="Action list sizes" src="https://user-images.githubusercontent.com/293280/121766871-c1b5e680-cb09-11eb-8835-bfec4c80e45a.png" />
+        </Flex>
+    </div>
+    <div>
+        <Heading fontSize={3}>Sizes</Heading>
+        <Flex as="p">Action list items support three different sizes: small, medium, and large. The small size is the default and most common option. Medium sizes work well for relaxed local navigation, while large sizes can support items that need more breathing room.</Flex>
+        <Flex as="p">Sizes only grow vertically. This behavior keeps the content aligned among items, and retains horizontal space for density.</Flex>
+        <Flex as="p">On touch devices, the large size is used at all times to ensure usability when tapping.</Flex>
+        <Flex as="p">Avoid mixing different sizes in the same list.</Flex>
```

Comment:
"Avoid mixing" or "Do not mix"?

Reply:
I think it's a message on the "caution" side instead of fully prohibitive. Do you think that'd work in this case?